tests/system/robot/chromeTests.py

```diff
@@ -649,3 +649,52 @@ def test_i10840():
 		nextActualSpeech,
 		"column 2  items"
 	)
+
+
+def test_mark_browse():
+	_chrome.prepareChrome(
+		"""
+		<div>
+			<p>The word <mark>Kangaroo</mark> is important.</p>
+		</div>
+		"""
+	)
+	actualSpeech = _chrome.getSpeechAfterKey('downArrow')
+	_asserts.strings_match(
+		actualSpeech,
+		"The word  highlighted  Kangaroo  out of highlighted  is important."
+	)
+	# Test moving by word
+	actualSpeech = _chrome.getSpeechAfterKey("numpad6")
+	_asserts.strings_match(
+		actualSpeech,
+		"word"
+	)
+	actualSpeech = _chrome.getSpeechAfterKey("numpad6")
+	_asserts.strings_match(
+		actualSpeech,
+		"highlighted  Kangaroo  out of highlighted"
+	)
+
+
+def test_mark_focus():
+	_chrome.prepareChrome(
+		"""
+		<div>
+			<p>The word <a href="#"><mark>Kangaroo</mark></a> is important.</p>

```

Comment:
when a link is focused, only the link's name is reported, not its inner content. This is pretty standard for any focusable elements, where the name is trusted over the inner content. 
Perhaps a test where the mark element is itself focusable could work, but neither of these are that real-world.

Reply:
A use case to consider: blogging platforms such as medium. The content is not editable, but viewers can see commonly highlighted parts of the article, this might contain a link: [here is a medium example](https://blog.medium.com/introducing-highlights-a4df69e8ed43), look for:
> this sentence is highlighted.